src/mono/netcore/sample/wasm/Directory.Build.props

```diff
@@ -0,0 +1,20 @@
+<Project>
+  <PropertyGroup>
+    <Configuration>Release</Configuration>
+    <OutputType>Exe</OutputType>
+
+    <RunAOTCompilation Condition="'$(RunAOTCompilation)' == ''">false</RunAOTCompilation>
+    <PublishTrimmed>true</PublishTrimmed>
+    <TrimMode>link</TrimMode>
+    <RunAnalyzers>false</RunAnalyzers>
+  </PropertyGroup>
+
+  <Import Project="..\Directory.Build.props"/>
+
+  <PropertyGroup>
+    <OutputPath>bin</OutputPath>
+    <WasmAppDir>$(MSBuildProjectDirectory)\bin\$(Configuration)\AppBundle\</WasmAppDir>
+  </PropertyGroup>
+
+  <Import Project="$(MonoProjectRoot)wasm\build\WasmApp.InTree.props" />
```

Comment:
Could this also be moved to the top? Or does it make more sense to keep it at the bottom?

Reply:
`*InTree.props` depends on runtime specific properties set by the inherited `Directory.Build.props`, like `$(ArtifactsBinDir)`, so this needs to be imported at the end.